lib/telephony/pinpoint/sms_sender.rb

```diff
@@ -39,9 +42,7 @@ def send(message:, to:, country_code:, otp: nil)
                 sms_message: {
                   body: message,
                   message_type: 'TRANSACTIONAL',
-                  origination_number: origination_number(country_code, sms_config, sender_id),
-                  sender_id: sender_id,
-                },
+                }.merge(sender_config),
```

Comment:
another option would be `{}.compact` which would remove keys with nil values 🤷  again six of one, half a dozen of the other